src/pages/_app.tsx

```diff
@@ -143,26 +142,3 @@ export default function App({
 		</QueryClientProvider>
 	)
 }
-
-App.getInitialProps = async ({
-	Component,
-	ctx,
-}: CustomAppContext<{ Component: ComponentType }>) => {
-	let pageProps = {}
-
-	if (Component.getInitialProps) {
-		pageProps = await Component.getInitialProps(ctx)
-	}
-
-	let host
-	if (ctx.req) {
-		host = ctx.req.headers.host
-	} else {
-		host = window.location.host
-	}
-
-	return {
-		pageProps,
-		host,
```

Comment:
The `host` value was only used during development by `getDeployedUrl` so it's low stakes to remove.

`Component.getInitialProps` is not ever defined to `pageProps` is always an empty `{}`

...therefore, `App.getInitialProps` is safe to remove entirely.